src/app/qgsoptions.cpp

```diff
@@ -553,6 +553,53 @@ QgsOptions::QgsOptions( QWidget *parent, Qt::WindowFlags fl, const QList<QgsOpti
     mKeepBaseUnitCheckBox->setChecked( false );
   }
 
+  // ***** 3D measurement settings *****
```

Comment:
Hmm do we need to have special configuration of decimal places, units, colors for 3D measure tool? I would suggest to reuse the 2D measure tool's configuration. It will keep the options simpler and it is also probably what people expect (not to have two different sets of options for measure tool)

Reply:
Agreed

Reply:
Ah ok, that makes sense. I will remove it.